packages/patternfly-4/react-table/src/components/Table/utils/index.d.ts

```diff
@@ -3,11 +3,12 @@ import { IRow, IExtra } from '../Table';
 
 export interface ISortable {
   className: string;
-  children: ReactNode
+  children: ReactNode;
 }
 
 export const sortable: (label: string, extra: IExtra) => ISortable;
 export const headerCol: () => { component: string };
 export const cellWidth: (width: string) => () => { className: string };
 export const expandable: (value: ReactNode, extra: IExtra) => ReactNode;
 export const isRowExpanded: (row: IRow, rows: Array<IRow>) => boolean | undefined;
+export { IExtra };
```

Comment:
@suomiy ok for now or did you have something else in mind?

Reply:
why is this necessary? `IExtra` should be already exported from `Table.d.ts` and `index.d.ts`

Reply:
```
ERROR in /Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-table/dist/js/components/Table/index.d.ts(18,53):
TS2305: Module '"/Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-table/dist/js/components/Table/utils/index"' has no exported member 'IExtra'.
```
can you try it? tested with :
```
    "@patternfly/patternfly": "1.0.240",
    "@patternfly/react-core": "2.9.0",
    "@patternfly/react-table": "1.3.2",
```

Reply:
ok, for temp fix has moved it to `index.d.ts` to match your inline-edit change for now:
https://github.com/patternfly/patternfly-react/pull/1227/files#diff-7d0e921725e74d478d1bee10116f6e8b